modules/linear_boltzmann_solvers/lbs_solver/io/flux_moments_io.cc

```diff
@@ -26,12 +26,10 @@ LBSSolverIO::WriteFluxMoments(
 
   const auto& grid = lbs_solver.Grid();
   const auto& discretization = lbs_solver.SpatialDiscretization();
-
   const auto& uk_man = lbs_solver.UnknownManager();
-  const auto nodes_only = UnknownManager::GetUnitaryUnknownManager();
 
   uint64_t num_local_cells = grid.local_cells.size();
-  uint64_t num_local_nodes = discretization.GetNumLocalDOFs(nodes_only);
+  uint64_t num_local_nodes = discretization.GetNumLocalNodes();
   uint64_t num_moments = lbs_solver.NumMoments();
   uint64_t num_groups = lbs_solver.NumGroups();
   uint64_t num_local_dofs = num_local_nodes * num_moments * num_groups;
```

Comment:
Done. I was originally hesitant to do this because we are writing a particular type to binary which has implications on the reader as well. Things look like they still work with `auto`.